bundles/CoreBundle/src/Command/ThumbnailsVideoCommand.php

```diff
@@ -141,25 +141,42 @@ protected function waitTillFinished(int $videoId, string|Asset\Video\Thumbnail\C
 
         // initial delay
         $video = Asset\Video::getById($videoId);
-        $thumb = $video->getThumbnail($thumbnail);
-        if ($thumb['status'] != 'finished') {
+        $thumb = $video?->getThumbnail($thumbnail);
```

Comment:
You do check in line 154 if the video was found too right?

Could we not skip it all together when the video was not found here?
The videoId won't change as far as I can see, so it would make sense if we skip it all together if the video was not found here instead of using the nullsafe operator here?

Reply:
you are right. I changed it to exit if the video was not found.

Reply:
Thank you!